src/main/java/com/itschool/eventmanagment/controllers/EventControler.java

```diff
@@ -2,20 +2,36 @@
 
 import com.itschool.eventmanagment.models.dtos.EventDTO;
 import com.itschool.eventmanagment.services.EventService;
+import jakarta.validation.constraints.NotNull;
+import org.springframework.format.annotation.DateTimeFormat;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.time.LocalDateTime;
+import java.util.List;
+
 @RestController
 public class EventControler {
     private final EventService eventService;
 
     public EventControler(EventService eventService) {
         this.eventService = eventService;
     }
+
     @PostMapping("/api/events")
-    public ResponseEntity<EventDTO> createEvent(@RequestBody EventDTO eventDTO){
+    public ResponseEntity<EventDTO> createEvent(@RequestBody EventDTO eventDTO) {
         return ResponseEntity.ok(eventService.createEvent(eventDTO));
     }
+
+    @GetMapping("/api/events")
+    public ResponseEntity<List<EventDTO>> getFilteredAndSortedEvents(@RequestParam @NotNull(message = "from date is mandatory") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) LocalDateTime from,
```

Comment:
Awesome job, good overall changes.
Just a question from my side: do we still need `LocalDateTimeSerializer`and `LocalDateTimeDeserializer` classes, or maybe you forgot to delete them? Besides that, everything seems to be ok.

Reply:
Also, please don't forget to fix the conflicts for this PR.

Reply:
LocalDateTimeSerializerand LocalDateTimeDeserializer  is need it for below endpoint. For filtering the events I've deleted as recomand and replace it with annotation. Love to use annotation since is so easy... If I can use annotation for POST I will be happy to do a fix later.
 
```
 @PostMapping("/api/events")
    public ResponseEntity<EventDTO> createEvent(@RequestBody EventDTO eventDTO) {
        return ResponseEntity.ok(eventService.createEvent(eventDTO));
    }
```